packages/web-components/src/components/ic-checkbox/ic-checkbox.tsx

```diff
@@ -25,40 +25,40 @@ import {
 })
 export class Checkbox {
   /**
-   * Set the checkbox to the checked state
+   * If `true`, the checkbox will be set to the checked state.
    */
   @Prop({ reflect: true, mutable: true }) checked?: boolean = false;
   /**
-   * Set the checkbox to the disabled state
+   * If `true`, the checkbox will be set to the disabled state.
    */
   @Prop() disabled?: boolean = false;
   /**
-   * Provide a label for the input
+   * The label for the input.
    */
   @Prop() label!: string;
   /**
-   * Provide a value for the input
+   * The value for the input.
    */
   @Prop() value!: string;
   /**
-   * Provide a name for the input. If not set when used in a checkbox group, the name will be based on the group name.
+   * The name for the input. If not set when used in a checkbox group, the name will be based on the group name.
    */
   @Prop() name: string;
   /**
-   * Provide a group label for the input
+   * The group label for the input.
    */
   @Prop() groupLabel: string;
   /**
-   * Provide a text prop for the dynamic text
+   * The text prop for the text displayed when dynamic.
```

Comment:
Doesn't quite read right to me - too many occurrences of "text"
```suggestion
   * The text to be displayed when dynamic.
```